packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -375,15 +310,12 @@ export class IndexedDbMutationQueue implements MutationQueue {
           control.done();
           return;
         }
-        const mutationKey = this.keyForBatchId(batchID);
+        const mutationKey = this.rangeForSingleBatchId(batchId);
         // Look up the mutation batch in the store.
-        // PORTING NOTE: because iteration is callback driven in the web,
-        // we just look up the key instead of keeping an open iterator
-        // like iOS.
         return mutationsStore(transaction)
-          .get(mutationKey)
-          .next(dbBatch => {
-            if (dbBatch === null) {
+          .loadAll(DbMutationBatch.userMutationsIndex, mutationKey)
```

Comment:
Can't we just use the batchId to read the batch directly without using the userMutationsIndex?  (same below)